File: JS Exams/01.  Basics Exam Collection/Basics Online Exam - 6 and 7 July 2019/2.FamiliTrip/2.FamiliTrip.js
function familyTrip(p) {
  let budget = +p[0]
  let nights = +p[1]
  let price = +p[2]
  let percent = +p[3]
  let subtotal = 0

  if (nights > 7) {
    subtotal = (price * 0.95) * nights //760
  } else {
    subtotal = (price) * nights //462 
  }

  let discount = budget * percent / 100
  let total = (subtotal + discount).toFixed(2)
  let finalSum = (budget - total).toFixed(2)


  if (finalSum < 0) {
    console.log(`${Math.abs(finalSum).toFixed(2)} leva needed.`);
  } else {
    s = Number(finalSum).toFixed(2)
    console.log(`Ivanovi will be left with ${s} leva after vacation.`);
  }
}

//familyTrip(['800.50', '8', '100', '2',])
//familyTrip(['500', '7', '66', '15',])

File: Python Exams/01. Basic Exam Collection/Basics Online Exam - 6 and 7 July 2019/2.FamiliTrip/2.FamiliTrip.py
budget, nights, price, percent = [float(input()) for _ in range(4)]
subtotal = 0

if nights > 7:
  subtotal = price * 0.95 * nights
else:
  subtotal = price * nights

discount = budget * percent / 100
total = subtotal + discount
final_sum = budget - total

if final_sum < 0:
    print(f'{abs(final_sum):.2f} leva needed.')
else:
    print(f'Ivanovi will be left with {final_sum:.2f} leva after vacation.')



'''
800.50
8
100
2
'''
